content/telegraf/v1.18/data_formats/output/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Telegraf output data formats
+description: Telegraf serializes metrics into output data formats for InfluxDB Line Protocol, JSON, Graphite, and Splunk metrics.
```

Comment:
Oh i don't know if we're listing all output data formats but if we are these need to be updated